cmd/list.go

```diff
@@ -40,6 +41,10 @@ func newListCommand(opts *dobiOptions) *cobra.Command {
 	flags.BoolVarP(
 		&listOpts.all, "all", "a", false,
 		"List all resources, including those without descriptions")
+	flags.BoolVarP(
+		&listOpts.groups, "groups", "g", false,
```

Comment:
What do you think about calling this flag `grouped` instead of `groups`?

Reply:
Ok, will use grouped